engine/src/main/resources/templates/MalwareDetection/detectedMalwareInstantEmailBody.html

```diff
@@ -0,0 +1,113 @@
+{#include MalwareDetection/insightsEmailBody}
+{#content-title}Malware Detection instant notification - {action.timestamp.toStringFormat()}{/content-title}
+{#content-body}
+<!-- Body section -->
+<tr>
+  <td class="rh-body">
+    <table class="rh-content" role="presentation" align="center" border="0" cellpadding="0" cellspacing="0" width="100%">
+      <tr>
+        <td>
+          <table class="rh-content" role="presentation" align="center" border="0" cellpadding="0" cellspacing="0" width="100%">
+            <tbody>
+
+            <!-- Number of new matched signatures section -->
+            <tr>
+              <td>
+                <table class="rh-content" role="presentation" align="center" border="0" cellpadding="0" width="100%">
+                  <tr>
+                    <td class="rh-card rh-m-danger" style="width: 10%;">
+                      <img src="https://console.redhat.com/apps/frontend-assets/email-assets/lightning-bolt-light.png" alt="" width="30">
+                    </td>
+                    <td class="rh-card rh-m-danger rh-card-metrics" style="padding-left: 0; width: 90%">
+                      <p style="margin-bottom: 0; font-weight: bold; !important">New matches</p>
+                      <p class="rh-metric__count">{action.events[0].payload.matched_rules.size()}</p>
+                    </td>
+                  </tr>
+                </table>
+              </td>
+            </tr>
+            <!-- End number of new matched signatures section -->
+
+            <!-- Empty row to space out content -->
+            <tr>
+              <td class="rh-body rh-multi-column">
+                <table class="rh-content" role="presentation" align="center" border="0" cellpadding="0" width="100%">
+                  <tbody><tr><td></td></tr></tbody>
+                </table>
+              </td>
+            </tr>
+            <!-- end empty row -->
+
+            <!-- User message section -->
+            <tr>
+              <td class="rh-content__block">
+                <p>
+                  Hi {user.firstName},
+                  <br>
+                  <br>
+                  Your system <a target="_blank" href="https://console.redhat.com/beta/insights/malware/systems/{action.events[0].payload.host_id}">{action.events[0].payload.host_name}</a>
```

Comment:
The `href` link will always redirect to prod while you may want a different link for stage. If you apply this suggestion, then the host will dynamically change depending on the environment where the email is generated:
```suggestion
                  Your system <a target="_blank" href="{environment.url}/beta/insights/malware/systems/{action.events[0].payload.host_id}">{action.events[0].payload.host_name}</a>
```

Reply:
Nice.  I've applied the change